client/reducers/userReducer.js

```diff
@@ -0,0 +1,17 @@
+import * as types from './../actions/actionTypes';
+import initialState from './initialState';
+
+export default (state = initialState.users, action) => {
+  switch (action.type) {
+    case types.GET_ALL_USERS_SUCCESS:
+      return action.users;
+
+    case types.CREATE_USER_SUCCESS:
+      return [
+        ...state,
+        Object.assign({}, action.user)
```

Comment:
What is the use of Object.assign({}, action.document) here? DO you think it adds any real value here? I do not think so.